contrib/vagrant/provision-util.sh

```diff
@@ -154,6 +156,12 @@ os::provision::set-os-env() {
   ln -s ${origin_root}/contrib/completions/bash/* /etc/bash_completion.d/
 }
 
+os::provision::get-bin-output-path() {
```

Comment:
Good call.  I've seen it used and for the case of refactoring nodes-are-ready its perfect.  I'll slim this PR down and update the other provisioning code to in a separate PR.